src/pages/main/MainPage.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import Navbar from '../../features/Navbar/Navbar';
+import headerData from '../../data/headerData';
+import Footer from '../../features/Footer/Footer';
+import Beers from '../../features/Beers/Beers';
+import Contact from '../../features/Contact/Contact';
+
+const MainPage = () => (
+  <div>
+    <Helmet>
+      <title>
+        {headerData.name}
+        {' '}
+        | Beer Lovers Application
+      </title>
+      <meta name="description" content="Beer lovers application" />
+    </Helmet>
+    <Navbar />
+    <Beers />
+    <Contact />
+    <Footer />
+
+  </div>
```

Comment:
- [ ] s per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Layout%3A%20composition%20and%20space%20between%20elements.), For the home page, Please make the beers types shaped in a two-column grid and the background color is staggered like a chessboard to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/211004825-ac1155f3-2d1d-497e-8ce1-a9ef5269d201.png)  | ![image](https://user-images.githubusercontent.com/68796232/211004804-0dcfce1f-6599-4df7-a51f-0643c5fda17c.png)  |